core/src/test/scala/unit/kafka/zk/KafkaZkClientTest.scala

```diff
@@ -377,10 +496,23 @@ class KafkaZkClientTest extends ZooKeeperTestHarness {
   }
 
   @Test
-  def testDeleteTopicPathMethods() {
-    val topic1 = "topic1"
-    val topic2 = "topic2"
+  def testDeletePath(): Unit = {
+    val path = "/a/b/c"
+    zkClient.createRecursive(path)
+    zkClient.deletePath(path)
```

Comment:
not familiar with `zkClient,` but don't we want to assert the path was created successfully before the delete?